src/inputs/input/input.jsx

```diff
@@ -29,46 +50,133 @@ const propTypes = {
      * Deprecated prop. Please use `disable` instead.
      */
     disabled: PropTypes.bool,
+    /**
+     * Indicates that the input has an error.
+     */
     error: PropTypes.oneOfType([
         PropTypes.bool,
         PropTypes.string,
     ]),
+    /**
+     * An input can take on the size of its container.
+     */
     fluid: PropTypes.bool,
+    /**
+     * Tells the component whether to be in guide or no guide mode.
+     */
     guide: PropTypes.bool,
+    /**
+     * Optional icon to display inside the input.
+     */
     icon: PropTypes.oneOfType([
         PropTypes.shape({}),
         PropTypes.string,
     ]),
+    /**
+     * The id is used for the a label\'s for.
+     */
     id: PropTypes.string,
+    /**
+     * Format to appear on dark backgrounds.
+     */
     inverse: PropTypes.bool,
+    /**
+     * Changes the general behavior of the Text Mask component.
+     */
     keepCharPositions: PropTypes.bool,
+    /**
+     * Optional Label to display with the Input.
+     */
     label: PropTypes.string,
+    /**
+     * Position the label above or below the input.
+     */
     labelPosition: PropTypes.oneOf(['bottom', 'top']),
+    /**
+     * Supply any inline styles to the label.
+     */
     labelStyle: PropTypes.shape({}),
+    /**
+     * An icon input field can show that it is currently loading data.
+     */
     loading: PropTypes.bool,
+    /**
+     * Is an array that defines how the user input is going to be masked.
```

Comment:
Maybe:
```
Define a mask to aid in use input by providing either an array of characters or a function.
```

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
You definitely adapted my suggestion.  Unfortunately I just realized the grammar in my suggestion was not good. :-(
I'll fix it.

Reply:
Fixed in [d096331](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/d0963311e2b27dac3c4389f84ddda4db25266335).